tsconfig.json

```diff
@@ -26,7 +26,7 @@
     // "isolatedModules": true,                     /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
 
     /* Strict Type-Checking Options */
-    "strict": true,                                 /* Enable all strict type-checking options. */
+    // "strict": true,                                 /* Enable all strict type-checking options. */
```

Comment:
what is the reason to switch this config off?

Reply:
the main reason was the lack of types for the lib. But I adjusted the type definitions we already had